website/projects/map4sci/src/app/map/map.component.ts

```diff
@@ -189,6 +184,30 @@ export class MapComponent {
       box-shadow: 3px 3px 5px 4px #00000021;
   `;
 
+  get edges(): Edge[] {
```

Comment:
While this is working, its not a very 'angular' way to do it. Please explore other options. Probably the best option is to use OnPush changeDetection and the NgOnChanges functionality to respond to new null values by setting defaults instead.